### DISCUSSION TITLE
Watchface maker not working. Stuck on authorizing

### DISCUSSION THREAD

**Discussion by CorruptionVR** [zepp-health/zeppos-forum#315] [Q&A]
Hey, I am not able to log back into the https://watchface.zepp.com web app. I get stuck on https://watchface.zepp.com/unauthorized. I was wondering if this was just for me, or if this is happening to other people.

**Comment by silver-zepp** [comment]
Hey @glitch128. I've confirmed the issue and the cloud team will investigate it first thing Monday. Thanks for the report.

**Reply by CorruptionVR** [reply]
Nobody really reported this? Am I the only one using the web app?

**Reply by silver-zepp** [reply]
It's a recent bug, otherwise, it would have been fixed long ago. But yes, most of our watch face developers don't use the generator and manually craft their WFs from scratch. How to: [Reference 1](https://github.com/zepp-health/zeppos-samples/tree/620134bfdeac3443f9e65510faaa15970a6e641d/watchface/1.0), [Reference 2](https://docs.zepp.com/docs/watchface/api/hmUI/widget/IMG/).

**Reply by emiegna** [reply]
Thank you very much, silver-zepp

**Reply by silver-zepp** [reply]
The problem has been resolved, guys.

**Reply by kingfisherimperialist** [reply]
Nicee

**Comment by kingfisherimperialist** [comment]
Same thing. Didn't download my assets back from the constructor((
